mesonbuild/scripts/python_info.py

```diff
@@ -76,4 +90,5 @@ def links_against_libpython():
   'is_venv': sys.prefix != variables['base_prefix'],
   'link_libpython': links_against_libpython(),
   'suffix': suffix,
+  'limited_api_suffix': limited_api_suffix
```

Comment:
Please end this with a comma :D that way the next person to update this doesn't have to change two lines.

Although we are dumping json, this file isn't actually json, which means we can do sensible things like end an entry with a comma even when commas aren't mandatory -- biggest gripe against json, a trailing comma is outright illegal. :(

Reply:
Done!